packages/lablqml/lablqml.0.6.2/opam

```diff
@@ -9,29 +9,17 @@ tags: [ "gui" "ui" "qt" ]
 build: [
   ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH ./configure"]
       {os-distribution = "alpine" | os-distribution = "centos" | os-distribution = "fedora" }
-  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH make"]
-      {os-distribution = "alpine" | os-distribution = "centos" | os-distribution = "fedora" }
-  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH make demos"]
-      { (os-distribution = "alpine"  | os-distribution = "centos" | os-distribution = "fedora") & with-test }
-
   ["./configure"]
       { os-distribution != "alpine" & os-distribution != "centos" & os-distribution != "fedora" }
-  ["dune" "build" "-p" "lablqml,ppx_qt"]
-      { os-distribution != "alpine" & os-distribution != "centos" & os-distribution != "fedora" }
+  ["dune" "build" "-p" "lablqml,ppx_qt" "-j" jobs]
   [make "demos"]
```

Comment:
Have you removed `{with-test}` on purpose?

Reply:
oops indeed, my bad